### PR TITLE
PWGDQ: adding MC truth template for dilepton-track

### PR DESCRIPTION
**Pull request by lucamicheletti93**
Adding MC truth template for dilepton-track

### CODE REVIEW

File: PWGDQ/DataModel/ReducedInfoTables.h [hunk 1]
```diff
@@ -401,6 +401,32 @@ using Dilepton = Dileptons::iterator;
 using DileptonExtra = DileptonsExtra::iterator;
 using DimuonAll = DimuonsAll::iterator;
 
+// candidate information
+namespace reducedcandidate
```
Comment (iarsene) [inline PWGDQ/DataModel/ReducedInfoTables.h:405]: We should give a more descriptive name here. Something like "reducedtriplet" or "dileptonTrackCandidate"

File: PWGDQ/DataModel/ReducedInfoTables.h [hunk 2]
```diff
@@ -401,6 +401,32 @@ using Dilepton = Dileptons::iterator;
...
+DECLARE_SOA_COLUMN(Mass, mass, float);                //!
+DECLARE_SOA_COLUMN(Pt, pt, float);                    //!
+DECLARE_SOA_COLUMN(Eta, eta, float);                  //!
+DECLARE_SOA_COLUMN(Tauz, tauz, float);                //!
+DECLARE_SOA_COLUMN(Tauxy, tauxy, float);              //!
+DECLARE_SOA_COLUMN(Lz, lz, float);                    //! Longitudinal projection of decay length
+DECLARE_SOA_COLUMN(Lxy, lxy, float);                  //! Transverse projection of decay length
+} // namespace reducedcandidate
+
+DECLARE_SOA_TABLE(DQcandidates, "AOD", "RTDQCANDIDATE", //!
```
Comment (iarsene) [inline PWGDQ/DataModel/ReducedInfoTables.h:418]: Name of the table should be changed accordingly

File: PWGDQ/DataModel/ReducedInfoTables.h [hunk 3]
```diff
@@ -401,6 +401,32 @@ using Dilepton = Dileptons::iterator;
...
+                  reducedcandidate::McDecision,
+                  reducedcandidate::Mass,
+                  reducedcandidate::Pt,
+                  reducedcandidate::Eta,
+                  reducedcandidate::Tauz,
+                  reducedcandidate::Tauxy,
+                  reducedcandidate::Lz,
+                  reducedcandidate::Lxy);
+
+using DQcandidate = DQcandidates::iterator;
```
Comment (iarsene) [inline PWGDQ/DataModel/ReducedInfoTables.h:428]: same here

File: PWGDQ/Tasks/dqEfficiency.cxx [hunk 4]
```diff
@@ -837,9 +837,11 @@ struct AnalysisSameEventPairing {
 };
 
 struct AnalysisDileptonTrack {
+  Produces<aod::DQcandidates> dqcandidatesList;
   OutputObj<THashList> fOutputList{"output"};
   // TODO: For now this is only used to determine the position in the filter bit map for the hadron cut
   Configurable<string> fConfigTrackCuts{"cfgLeptonCuts", "", "Comma separated list of barrel track cuts"};
+  Configurable<bool> fConfigFlatTables{"cfgFlatTables", false, "Produce a single flat tables with all relevant information dilepton-track candidates"};
```
Comment (iarsene) [inline PWGDQ/Tasks/dqEfficiency.cxx:844]: Maybe name this option to something like "cfgFillCandidateTable" or something like that

File: PWGDQ/DataModel/ReducedInfoTables.h (hunk 1 above)
Comment (lucamicheletti93) [reply PWGDQ/DataModel/ReducedInfoTables.h:405]: Done!

File: PWGDQ/DataModel/ReducedInfoTables.h (hunk 2 above)
Comment (lucamicheletti93) [reply PWGDQ/DataModel/ReducedInfoTables.h:418]: Fixed

File: PWGDQ/DataModel/ReducedInfoTables.h (hunk 3 above)
Comment (lucamicheletti93) [reply PWGDQ/DataModel/ReducedInfoTables.h:428]: Fixed

File: PWGDQ/Tasks/dqEfficiency.cxx (hunk 4 above)
Comment (lucamicheletti93) [reply PWGDQ/Tasks/dqEfficiency.cxx]: Done!

**Review by iarsene — APPROVED** [review]

**Pull request merged**